lib/model/store.dart

```diff
@@ -341,6 +342,54 @@ class PerAccountStore extends ChangeNotifier with ChannelStore, MessageStore {
 
   final Map<int, User> users;
 
+  Map<String, Wildcard> wildcardsForNarrow(Narrow narrow) => Map.fromEntries(
+    _wildcardsForNarrow(narrow).map((w) => MapEntry(w.name, w)));
+
+  List<Wildcard> _wildcardsForNarrow(Narrow narrow) {
+    final isDmNarrow = narrow is DmNarrow;
+    final isChannelWildcardAvailable = account.zulipFeatureLevel >= 247; // TODO(server-9)
+    final isTopicWildcardAvailable = account.zulipFeatureLevel >= 188; // TODO(sever-8)
+    return [
+      Wildcard(
```

Comment:
Does this mirror the corresponding logic in zulip-mobile? (I haven't read this closely.) A link for comparison would be handy.

Reply:
It doesn't follow the zulip-mobile in the exact same way. Here's the web implementation: https://github.com/zulip/zulip-mobile/blob/main/src/autocomplete/WildcardMentionItem.js

Reply:
OK. Let's try to make sure it gets the same results, even if the code ends up organized differently. (There should be test cases in zulip-mobile that will be helpful to translate over, to verify that.)

Then also include a link to that zulip-mobile implementation in either a code comment or commit message. For the link, make it a GitHub permalink:
https://github.com/zulip/zulip-flutter/pull/692#discussion_r1674802255
https://github.com/zulip/zulip-flutter/pull/884#discussion_r1714682330
That'll be helpful for reviewers, as a point of comparison.